src/catalog/src/builtin.rs

```diff
@@ -1810,7 +1818,23 @@ pub static MZ_MAP_TYPES: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
     desc: RelationDesc::empty()
         .with_column("id", ScalarType::String.nullable(false))
         .with_column("key_id", ScalarType::String.nullable(false))
-        .with_column("value_id", ScalarType::String.nullable(false)),
+        .with_column(
+            "key_modifiers",
+            ScalarType::List {
+                element_type: Box::new(ScalarType::Int64),
+                custom_id: None,
+            }
+            .nullable(true),
+        )
```

Comment:
I can't fully remember the reason right now, but I'm pretty sure adding a new column in the middle of existing columns (i.e. not at the end) is a breaking change. That's probably fine, but let's remember to check for usages of this table in the next release.

Reply:
I don't think the order matters that much. I moved the key column after the existing ones.